src/main/java/frc/robot/subsystems/Scoring.java

```diff
@@ -51,23 +48,20 @@ public static Scoring getInstance() {
 
     public Scoring() {
         super(new PIDController(ScoringConstants.kP, ScoringConstants.kI, ScoringConstants.kD));
+
         rollerMotor.restoreFactoryDefaults();
         rightScoringElevatorMotor.restoreFactoryDefaults();
         leftScoringElevatorMotor.restoreFactoryDefaults();
-        leftLoadingMotor.restoreFactoryDefaults();
-        rightLoadingMotor.restoreFactoryDefaults();
+
         rollerMotor.setIdleMode(IdleMode.kCoast);
         rightScoringElevatorMotor.setIdleMode(IdleMode.kCoast);
         leftScoringElevatorMotor.setIdleMode(IdleMode.kCoast);
-        leftLoadingMotor.setIdleMode(IdleMode.kCoast);
-        rightLoadingMotor.setIdleMode(IdleMode.kCoast);
+
         rightScoringElevatorMotor.follow(leftScoringElevatorMotor);
-        rightLoadingMotor.follow(leftLoadingMotor);
+
         rollerMotor.burnFlash();
         rightScoringElevatorMotor.burnFlash();
         leftScoringElevatorMotor.burnFlash();
-        leftLoadingMotor.burnFlash();
-        rightLoadingMotor.burnFlash();
     }
 
     @Override
```

Comment:
I would say we should stay standard with our naming. Rename the output parameter to speed and adjust the parameter's value within the constructor when clamping instead of creating a new variable.

Reply:
"useOutput" is a method name that is not specific enough.

Reply:
Should there be a reference to "super?"

Reply:
Should there be a reference to "super?"

Reply:
"useOutput" is a method name that is not specific enough.

Reply:
`useOutput` is a function that is overridden from `PIDSubsystem` so we unfortunately don't have a choice